core/exec.c

```diff
@@ -2573,7 +2573,7 @@ static lai_api_error_t lai_exec_parse(int parse_mode, lai_state_t *state) {
         if (lai_current_instance()->trace & LAI_TRACE_OP){
             char* path = lai_stringify_amlname(&amln);
```

Comment:
Maybe use `LAI_CLEANUP_FREE_STRING` here instead of freeing it manually

Reply:
Yeah I considered that too, and couldn't sleep because I hoped the PR wouldn't be merged before I made that change.